zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1272,6 +1272,44 @@ paths:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
 
+  /elements/{elementKey}/variabels:
```

Comment:
```suggestion
  /elements/{elementKey}/variables:
```

❌ Spelling error. From my very non-Java-engineer perspective, I _think_ this is spelled correctly in the corresponding Java class, so the spelling correction is not needed elsewhere.